### PR TITLE
Feature/project setup

### PR DESCRIPTION
**Pull request by Prangula**

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -45,4 +50,34 @@ dependencies {
...
+    implementation("androidx.navigation:navigation-ui-ktx:$navVersion")
+    implementation(platform("org.jetbrains.kotlin:kotlin-bom:1.8.0"))
+    implementation(kotlin("script-runtime"))
+    // Retrofit
+    implementation("com.karumi:dexter:6.0.2")
+    implementation("com.squareup.retrofit2:retrofit:2.9.0")
+    implementation("com.squareup.retrofit2:converter-gson:2.9.0")
+    // Glide
+    implementation("com.github.bumptech.glide:glide:4.11.0")
+    annotationProcessor("com.github.bumptech.glide:compiler:4.11.0")
```
Comment (etozheluka) [inline app/build.gradle.kts]: i think you dont need glide compiler 
image

File: app/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -2,4 +2,13 @@
 <resources>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="yellowPrimary">#F5C518</color>
+    <color name="neutralBlack">#080808</color>
+    <color name="neutralDarkestGrey">#1C1C1C</color>
```
Comment (ninikokra) [inline app/src/main/res/values/colors.xml]: please rename the colors as they are in Figma

File: app/src/main/res/drawable/caution_logo.xml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ninikokra) [inline app/src/main/res/drawable/caution_logo.xml]: rename this with ic_caution
all icons should start with ic_name and all backgrounds should start with bkg_name

File: app/src/main/res/drawable/imdb_logo.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (ninikokra) [inline app/src/main/res/drawable/imdb_logo.xml]: same here ic_logo_splash_screen or smthg like that

File: app/src/main/res/drawable/no_results_logo.xml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="106dp"
+    android:height="106dp"
+    android:viewportWidth="106"
```
Comment (ninikokra) [inline app/src/main/res/drawable/no_results_logo.xml]: also here

File: app/src/main/java/com/imdbmovieapp/ImdbMovieActivity.kt [hunk 6]
```diff
@@ -0,0 +1,11 @@
+package com.imdbmovieapp
+
+import android.os.Bundle
+import androidx.appcompat.app.AppCompatActivity
+
+class ImdbMovieActivity : AppCompatActivity() {
```
Comment (ninikokra) [inline app/src/main/java/com/imdbmovieapp/ImdbMovieActivity.kt]: i dont like this name, you can try to simplify it with just MoviesActivity? imo its more readable

**Review by ninikokra — CHANGES_REQUESTED** [review]

File: app/src/main/res/values/colors.xml [hunk 7]
```diff
@@ -2,4 +2,13 @@
 <resources>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="yellowPrimary">#F5C518</color>
+    <color name="neutral01Black">#080808</color>
+    <color name="neutral02DarkestGrey">#1C1C1C</color>
+    <color name="neutral03DarkGrey">#5D5D5D</color>
+    <color name="neutral04Grey">#808080</color>
+    <color name="neutral05LightGrey">#A5A5A5</color>
```
Comment (ninikokra) [inline app/src/main/res/values/colors.xml]: better to write neutral_04_Grey, neutral_05_light_grey it's more readable

**Review by ninikokra — CHANGES_REQUESTED** [review]

**Review by ninikokra — APPROVED** [review]

**Review by Davit-Tsiskarashvili — APPROVED** [review]

**Review by giorgimurgulia-space — APPROVED** [review]

**Pull request merged**